### ISSUE TITLE
Enchance traffic shaping

### ISSUE THREAD

**Issue by gsanchietti** [NethServer/dev#5113]
Current implementation of traffic shaping has 2 limitations:

- it supports only rules based on destination port or source ip
- the web interface is not integrated with firewall configuration, and often users forget to set the bandwidth of red interfaces 

Proposed changes:

- [x] Allow the creation of complex rules also with nDPI support.
  Example: during the office hours, facebook traffic has low priority
- [x] Expand the "Firewall rules" page: add the "low priority" and "high priority" actions on rule creation
- [x] Move bandwidth configuration of red network interface inside the "Network" page
- [x] Remove the "Traffic shaping" page

**Comment by DavidePrincipi** [comment]
In nethserver-testing:
nethserver-firewall-base-3.0.4-1.41.g662aa08.ns7.noarch.rpm
nethserver-firewall-base-ui-3.0.4-1.41.g662aa08.ns7.noarch.rpm

**Comment by dz00te** [comment]
just a note to other testers: be sure to update also
nethserver-httpd-admin-2.0.2-1.1.ga8abb48.ns7.noarch.rpm

**Comment by gsanchietti** [comment]
Back on QA:
- nethserver-firewall-base-3.0.4-1.50.g08420d6.ns7.noarch.rpm
- nethserver-firewall-base-ui-3.0.4-1.50.g08420d6.ns7.noarch.rpm
- nethserver-httpd-admin-2.0.2-1.1.ga8abb48.ns7.noarch.rpm
- nethserver-base-3.0.8-1.4.ge256e3b.ns7.noarch.rpm

Before testing, see https://github.com/NethServer/nethserver-firewall-base/blob/master/README.rst#traffic-shaping

**Comment by filippocarletti** [comment]
I think I have tested many (if not all) combinations of traffic with different priorities.
The only configuration that I found not working as expected is traffic originating from the firewall itself but only when multi wan is configured with more than one provider.
It may be due to a shorewall bug that is being worked on, we will probably have a fix in the near future, simply updating shorewall to 5.0.12.
Technically, the tcout mangle chain is not entered because packets are already marked by multi wan in PREROUTING.
